pkg/controller/configmap/configmap_controller.go

```diff
@@ -307,23 +310,34 @@ func (r *ReconcileConfigMap) Reconcile(request reconcile.Request) (reconcile.Res
 	}
 
 	// Get nsx-secret and lb-secret
-	var opNsxSecret *corev1.Secret
-	if optionInConfigMap(instance, "nsx_v3", "nsx_api_cert_file") {
-		opNsxSecret = &corev1.Secret{}
-		err = r.client.Get(context.TODO(), types.NamespacedName{Namespace: watchedNamespace, Name: operatortypes.NsxSecret}, opNsxSecret)
+	opNsxSecret := &corev1.Secret{}
+	err = r.client.Get(context.TODO(), types.NamespacedName{Namespace: watchedNamespace, Name: operatortypes.NsxSecret}, opNsxSecret)
```

Comment:
Is it safe to remove `if optionInConfigMap(instance, "nsx_v3", "nsx_api_cert_file") {...` before L314?

Reply:
yes, I think so, there should be no nsx_api_cert_file option any more since these options are removed from configmap.